### PR TITLE
Added a MultiCPU SLURM example using Accelerate Launch and MPIRun

### PR DESCRIPTION
**Pull request by okhleif-IL**
# What does this PR do?
This PR adds a multicpu SLURM example in the `examples/slurm` folder using Accelerate and MPIRun. This script can run a multicpu SLURM training example on the `complete_nlp_example.py` script.

This PR also fixes a minor typo in all other SLURM example .sh scripts regarding activating the environment.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/accelerate/blob/main/CONTRIBUTING.md#submitting-a-pull-request-pr),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/accelerate/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/accelerate/tree/main/docs#writing-documentation---specification).
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: src/accelerate/utils/launch.py [hunk 1]
```diff
@@ -250,7 +259,6 @@ def prepare_multi_gpu_env(args: argparse.Namespace) -> Dict[str, str]:
         current_env["FSDP_USE_ORIG_PARAMS"] = str(args.fsdp_use_orig_params).lower()
         current_env["FSDP_CPU_RAM_EFFICIENT_LOADING"] = str(args.fsdp_cpu_ram_efficient_loading).lower()
         current_env["FSDP_SYNC_MODULE_STATES"] = str(args.fsdp_sync_module_states).lower()
-        current_env["FSDP_ACTIVATION_CHECKPOINTING"] = str(args.fsdp_activation_checkpointing).lower()
```
Comment (muellerzr) [inline src/accelerate/utils/launch.py:253]: Why are we removing this?

**Review by muellerzr — COMMENTED** [review]
Thanks! Overall LG2M, bar one nit

File: src/accelerate/utils/launch.py (hunk 1 above)
Comment (okhleif-IL) [reply src/accelerate/utils/launch.py]: Must've been a mistake--fixed in my latest commit

Comment (okhleif-IL) [reply src/accelerate/utils/launch.py]: Thanks

**Review by muellerzr — APPROVED** [review]
Thanks!

**Pull request merged**